app/scripts/lib/channels/duplex.js

```diff
@@ -155,7 +155,7 @@ define([
      * @param {Object} [data]
      * @return {String}
      */
-    createMessageId: function (command, data) { //jshint ignore: line
+    createMessageId: function (command, data) {
```

Comment:
Does eslint check for unused variables? These are both unused in the traditional sense, but are used by the docs.